src/Raven.Server/Commercial/LetsEncrypt/SettingsZipFileHelper.cs

```diff
@@ -355,57 +669,36 @@ public static string CreateReadmeText(string nodeTag, string publicServerUrl, bo
                 "If you encounter this bug, use the RavenCli command 'generateClientCert' to create a new certificate with a password." +
                 Environment.NewLine +
                 "For more information on this workaround, read the security documentation in 'ravendb.net'." +
-                Environment.NewLine;
-
-        str +=
-            Environment.NewLine +
-            "It is recommended to generate additional certificates with reduced access rights for applications and users." +
-            Environment.NewLine +
-            "This can be done using the RavenDB Studio, in the 'Manage Server' > 'Certificates' page." +
-            Environment.NewLine;
-
-        if (isCluster)
-        {
-            str +=
-                Environment.NewLine +
-                "You are setting up a cluster. The cluster topology and node addresses have already been configured." +
-                Environment.NewLine +
-                "The next step is to download a new RavenDB server for each of the other nodes." +
-                Environment.NewLine +
-                Environment.NewLine +
-                "When you enter the setup wizard on a new node, please choose 'Continue Existing Cluster Setup'." +
-                Environment.NewLine +
-                "Do not try to start a new setup process again in this new node, it is not supported." +
-                Environment.NewLine +
-                "You will be asked to upload the zip file which was just downloaded." +
-                Environment.NewLine +
-                "The new server node will join the already existing cluster." +
                 Environment.NewLine +
+                "It is recommended to generate additional certificates with reduced access rights for applications and users." +
                 Environment.NewLine +
-                "When the wizard is done and the new node was restarted, the cluster will automatically detect it. " +
-                Environment.NewLine +
-                "There is no need to manually add it again from the studio. Simply access the 'Cluster' view and " +
-                Environment.NewLine +
-                "observe the topology being updated." +
+                "This can be done using the RavenDB Studio, in the 'Manage Server' > 'Certificates' page." +
                 Environment.NewLine;
-        }
 
         return str;
     }
-    
-    private static string IpAddressToUrl(string address, int port)
+
+    internal static string IpAddress(string address, int port)
```

Comment:
Name should describe what it does e.g.
```suggestion
    internal static string IpAddressToUrl(string address, int port)
```

Reply:
Actually there's already very similar one like that below.

Reply:
Done